compiler/src/main/java/com/dylibso/chicory/compiler/internal/WasmAnalyzer.java

```diff
@@ -40,6 +54,18 @@ final class WasmAnalyzer {
     private final List<ValType> tableTypes;
     private final int functionImports;
 
+    private static class TryCatchBlock {
```

Comment:
I find the previous encoding as a CompilerOpCode with 3 operands for the labels better suited